app/controllers/v0/education_benefits_claims_controller.rb

```diff
@@ -11,6 +11,17 @@ def create
       render(json: education_benefits_claim)
     end
 
+    # TODO: This is hidden behind a flag in the routes, but it's duplicated here for some
+    # defense in depth. This functionality should not be included once EducationForm is
+    # released in production, but rather should be available in a dedicated admin interface
+    def show
+      (redirect_to(root_path) && return) if Rails.env.production? && ENV['EDU_FORM_SHOW'] != 'true'
```

Comment:
would it be better to only include the route if Rails.env.production? && ENV['EDU_FORM_SHOW']

and then just do a standard - `redirect_to root_path and return`